py/makeqstrdefs.py

```diff
@@ -77,6 +74,10 @@ def cat_together():
         pass
     if old_hash != new_hash:
         print("QSTR updated")
+        try:
+            os.remove(args.output_file)
```

Comment:
Indeed, I put that in the commit message but I'll add a comment here as well, and I'll address your other remarks as well.